plaso/cli/status_view.py

```diff
@@ -317,6 +318,9 @@ def PrintExtractionStatusHeader(self, processing_status):
     self._output_writer.Write(
         'Source type\t: {0:s}\n'.format(self._source_type))
 
+    if self._artifact_filters:
+      self._output_writer.Write('Artifact filters \t: {0:s}\n'.format(
```

Comment:
no need for space after `filters`:
'Artifact filters \t: {0:s}\n

either:
'Artifact filters\t\t: {0:s}\n
'Artifact filters\t: {0:s}\n

Reply:
Done